### DISCUSSION TITLE
Naming conflicts "Lectio11", "Lectio93", "Lectio94"

### DISCUSSION THREAD

**Discussion by FAJ-Munich** [DivinumOfficium/divinum-officium#2752] [Developers corner]
Dear all,

Given the heavy use of cross-referencing from the Monastic version to the Roman, I've realised problems may arrise where the Monastic version finds a "Lectio11" in the Roman Commune (C4, C5, C5a, and C6) which is actually intended to be a variant of a "Lectio1". Since the code also makes use of the convention that "Lectio93" and "Lectio94" are used for commemorations and contractions of saints' legends, similiar conflicts can arise where in the Commune (or CommuneM) various further Nocturn 3 lessons are provided (at the moment: C2, C4, and C5a).

I am happy to contribute and resolve this issue. Given I have only started recently, nonetheless, may I humbly ask for guidance which route to take: 

- Option 1: additional lessons are renamed with a clear distinction of their place, e.g, Lectio11, Lectio21, and Lectio31 in C4 become Lectio1loco2, Lectio2,loco2, etc.; and Lectio73, Lectio83, and Lectio93 become Lectio7loco4, Lectio8loco4, and Lectio9loco4 respectively
- Option 2: the addional lessons are put in distinct files of the Commune folder with respective cross-reference, e.g. a new file C4loco2.txt that contains hashes Lectio1 through Lectio 3 and Lectio7 through Lectio9 with their respective content

Since it may happen that in Nocturn 1 we have to direct to some additional lessons and in Nocturn2 or 3 to some lessons from a different "aliæ loco" ...., I personally would prefer the former as it less invasive.

Thanks for your advice!
Best regards,
F. from Munich

**Comment by gregordick** [comment]
From a quick look at the code, I don't think it will ever look up `Lectio11` directly in the Roman usages, and that it's only used in the `@` cross-references.  As such, you should just be able to rename the `Lectio11` sections in the commons and the adjust the `@` references accordingly.  If I understand correctly, this is what you were proposing in Option 1.
